packages/web-pkg/src/components/AppBar/AppBar.vue

```diff
@@ -112,6 +112,7 @@ import { storeToRefs } from 'pinia'
 import { RouteLocationRaw } from 'vue-router'
 
 export default defineComponent({
+  name: 'AppBar',
```

Comment:
Fyi the names are needed for some components to not end up as an `anonymous-stub` when used by a component that now uses script setup. This will eventually be obsolete once we refactor those components to script setup as well.